### DISCUSSION TITLE
SL opens on wrong screen (V10.5.2 @ TB 128.1.1)

### DISCUSSION THREAD

**Discussion by MaSiegmund** [Extended-Thunder/send-later#725] [Q&A]
Since a few days the SL-Window does not open on the screen where TB is open. Simply moving the window does not help. Next time the SL-Windows opens on the other screen again. Are there settings for this somewhere?

**Comment by jikamens** [comment]
I have absolutely no control over where Thunderbird decides to put the window.
I would like to attach the window to the compose window so it is always in the right place, but I can't do that until this is fixed: https://github.com/Extended-Thunder/send-later/issues/663
